tests/group/group_result_samples/add_group_result_error.json

```diff
@@ -4,14 +4,17 @@
       "name": "TESTGRPP0",
       "operation": "set",
       "requestId": "GroupRequest",
-      "error": {
-        "errorFunction": 10,
-        "errorCode": 2000,
-        "errorReason": 68,
-        "errorMessage": "Invalid attribute value specified.",
-        "errorOffset": 149,
-        "textInError": "name"
-      }
+      "commands": [
+        {
+          "safReturnCode": 10,
+          "returnCode": 2000,
+          "reasonCode": 68,
+          "image": "<securityrequest xmlns=\"http://www.ibm.com/systems/zos/saf\" xmlns:racf=\"http://www.ibm.com/systems/zos/racf\"><group name=\"TESTGRPP0\" operation=\"set\" requestid=\"GroupRequest\" /></securityrequest>",
+          "messages": [
+            "Invalid attribute value specified.\nText in Error: 'name'\nApproximate Offset of Text in Error: '149'\nPlease note that for any text in error, redacted values may skew offset calculations."
```

Comment:
I think these messages need to be split into separate lines:

```json
          "messages": [
            "Invalid attribute value specified.",
            "Text in Error: 'name'",
            "Approximate Offset of Text in Error: '149'",
            "Please note that for any text in error, redacted values may skew offset calculations."
          ]
```

Also, I don't know if it makes sense to have the xml elements as additional json keys like the old format so that it is easier to parse maybe:

```json
      "commands": [
        {
          "safReturnCode": 10,
          "returnCode": 2000,
          "reasonCode": 68,
          "image": ...,
          "messages": [
            "Invalid attribute value specified.",
            "Please note that for any text in error, redacted values may skew offset calculations."
          ]
          "errorOffset": 149,
          "textInError": "name"
        }
      ]
```

Reply:
I could be open to leaving these in json keys rather than the message, or maybe both. I kept them in messages to standardize things as RACF's "errors" will surface them in MESSAGES rather than something as simple to parse as this. Open to further discussion. Once we have a defined solution, I can change this to do separate messages.

Reply:
Makes sense, change made.

Reply:
XML format of these errors reverted back to their original format, but are now included with `DownstreamFatalError` instead of `SecurityRequestError`.